.eslintcache

```diff
@@ -0,0 +1 @@
+[{"/Users/mcsurf/craft academy/dream_time_news/dream_time_news_client_admin/src/reportWebVitals.js":"1","/Users/mcsurf/craft academy/dream_time_news/dream_time_news_client_admin/src/App.jsx":"2"},{"size":362,"mtime":1607544839292,"results":"3","hashOfConfig":"4"},{"size":150,"mtime":1607546146265,"results":"5","hashOfConfig":"4"},{"filePath":"6","messages":"7","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"11bndjj",{"filePath":"8","messages":"9","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"/Users/mcsurf/craft academy/dream_time_news/dream_time_news_client_admin/src/reportWebVitals.js",[],"/Users/mcsurf/craft academy/dream_time_news/dream_time_news_client_admin/src/App.jsx",[]]
```

Comment:
This one should be ignored by git

Reply:
done

Reply:
You need to clear the cache

Reply:
noted. we deleted it and deleted it from the gitignore to push the changes and then we added it back to the gitignore